app/containers/TracksContainer/index.js

```diff
@@ -1,5 +1,159 @@
-import React from 'react';
+import React, { memo, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { createStructuredSelector } from 'reselect';
+import styled from 'styled-components';
+import { tracksContainerCreators } from './reducer';
+import tracksContainerSaga from './saga';
+import { selectTrackName, selectTracksContainer, selectTracksData } from './selectors';
+import { debounce, get, isEmpty } from 'lodash';
+import { Input, Card, Skeleton } from 'antd';
+import T from '@components/T';
+import For from '@app/components/For';
+import If from '@app/components/If';
+import TrackComponent from '@app/components/TrackComponent/index';
 
-export default function TracksContainer() {
-  return <div> TracksContainer page </div>;
+const { Search } = Input;
+
+const TitleCard = styled(Card)`
+  && {
+    margin: 10px 0;
+    max-width: ${props => props.maxwidth}px;
+    color: ${props => props.color}px;
+  }
+`;
+
+const Container = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    max-width: ${props => props.maxwidth}px;
+    width: 100%;
+    margin: 0 auto;
+    padding: ${props => props.padding}px;
+  }
+`;
+
+export function TracksContainer({
+  maxwidth,
+  padding,
+  tracksError,
+  tracksData,
+  trackName,
+  intl,
+  dispatchTrackNames,
+  dispatchClearTracksData
+}) {
+  const [loading, setLoading] = useState(false);
```

Comment:
Instead of this, can we have the tracksLoading state in the reducer and check for loading state. Just like how we are checking if there is any error or not from tracksError that selects the error state from the reducer

Reply:
Updated the changes